assessments/0009-iin-toto.md

```diff
@@ -0,0 +1,292 @@
+
+**Content creation processes**
+
+- The [read-the-docs site](https://in-toto.readthedocs.io/en/latest/) has an [Edit the Doc button](https://github.com/in-toto/in-toto/blob/develop/doc/source/index.rst). Instead of pointing directly to the source files, it should point to a page with instructions for [doc contributors](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md), and a link to the [governance policies](https://github.com/in-toto/community/blob/main/GOVERNANCE.md).
+
+Material explicitly addressed to documentation contributions and standards should be added, and accessible from the top-level doc roadmap as well as from the existing contributor pages.
+- Procedures for creating and maintaining docs need to be documented.
+   - The process should include a policy for where reference docs live in the repo structure and how docs for different implementations are distinguished and identified.
+- Policy and procedures for documenting implementations need to be decided and published for contributors.
+   - Reference documentation for additional implementations should be actively solicited from the contributor community.
+
+## Contributor documentation
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     |     |  x  |     |
+| Beginner friendly issue backlog           |     |  x  |     |     |     |
+| “New contributor” getting started content |     |  x  |     |     |     |
+| Project governance documentation          |     |     |     |  x  |     |
+
+Scale:
+
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+**Comments**
+
+- **Communication methods documented:** [Contact channels list](https://in-toto.io/contact/) available from home page under **Community > Contact**.
+
+- **Beginner friendly issue backlog:** "Good First Issue" label is available and used for PRs for specific implementations. Not in much general use.
+
+- **“New contributor” getting started content:** [Community README](https://github.com/in-toto/community/blob/main/README.md) points to "Good First Issues" list for specific repos. Can't find explicit instructions for new contributors, or any first issues for other kinds of contribution (such as docs).
+
+- **Project governance documentation:** Clearly described and discoverable on [GOVERNANCE page](https://github.com/in-toto/community/blob/main/GOVERNANCE.md).
+
+**RECOMMENDATIONS**
+
+**Communication methods documented**
+
+- Since most of the documentation is in GitHub, rather than on the web site, the [Contact channels list](https://in-toto.io/contact/) could be usefully added to the READMEs for the [main](https://github.com/in-toto/in-toto) and [community](https://github.com/in-toto/community) repos.
```

Comment:
Should this be surfaced onto the website? I know that things like CONTRIBUTING often live in the repo, but there may be value in having these sorts of things on the website (I'm not too concerned about dulpication in this case as things are not likely to change once set.)